.gitignore

```diff
@@ -32,5 +34,3 @@ nohup\.out
 react-prototype/dist/*
 react-prototype/node_modules/*
 react-prototype/npm-debug\.log
-.idea/*
-!app/\.idea/codeStyleSettings.xml
```

Comment:
gitignore changes unrelated to this ticket, but my IDE was throwing out a few extra files I didn't want to commit. I noticed while I was doing it that we had a couple of pointless rules:

* `.idea/*`: There is no .idea folder in the root, and we don't want to exclude the whole .idea folder, just individual files
* `!app/\.idea/codeStyleSettings.xml`: since we're not excluding the whole folder, there's no need to re-include this one file.